### PR TITLE
Update view commands in UG

### PR DESCRIPTION
**Pull request by BlopApple**
Update the UG for the following view commands:
* `view --today`
* `view DATE`
* `view -p --all`
* `view -p`

The order was kept as it was before the change in #380.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -667,69 +667,74 @@ _Add screenshot here_
 
 <br>
 
-### Listing all patients for today: `view` `--today`
+### Listing all tasks for today: `view` `--today`
 
-Shows a list of all patients with tasks due today.
+You can view the list of tasks that are due today using the `view` command with the special flag `--today`.
 
 Format: `view --today`
```
Comment (izzahaj) [inline docs/UserGuide.md:674]: Would be good to bold the command word for consistency. (most of the other features have it)
```suggestion
Format: **`view`**`--today`
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -667,69 +667,74 @@ _Add screenshot here_
...
+_<div align="center"> Patient and task list displayed after running the `view --today` command </div>_
+
 <br>
 
 ### Listing all tasks: `view` `-p` `--all`
 
-Shows a list of all tasks to be completed.
+You can view the list of tasks for all patients using the `view` command with the special flag `--all`.
 
 Format: `view -p --all`
```
Comment (izzahaj) [inline docs/UserGuide.md:693]:
```suggestion
Format: **`view`**`-p --all`
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -667,69 +667,74 @@ _Add screenshot here_
...
+_<div align="center"> Patient and task list displayed after running the `view -p --all` command </div>_
 
 <br>
 
 ### Viewing all tasks of a patient: `view` `-p`
 
-Shows all the tasks that are associated with the specified patient.
+You can view the list of tasks for a patient using the `view` command.
 
 Format: `view -p PATIENT_INDEX`
```
Comment (izzahaj) [inline docs/UserGuide.md:708]:
```suggestion
Format: **`view`**`-p PATIENT_INDEX`
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -667,69 +667,74 @@ _Add screenshot here_
...
+_<div align="center"> Patient and task list displayed after running the `view -p 2` command </div>_
 
 <br>
 
 ### Listing all tasks for a particular day: `view`
 
-Shows a list of all tasks on a particular day.
+You can view the list of tasks for a particular day using the `view` command.
 
 Format: `view DATE`
```
Comment (izzahaj) [inline docs/UserGuide.md:723]:
```suggestion
Format: **`view`**`DATE`
```

**Review by izzahaj — CHANGES_REQUESTED** [review]
Some formatting inconsistencies. Looks ok otherwise

**Review by rama-pang — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 5]
```diff
@@ -667,69 +667,74 @@ _Add screenshot here_
...
-_Add screenshot here_
+![result for `view -p --all`](images/viewAllTaskResult.png)
+_<div align="center"> Patient and task list displayed after running the `view -p --all` command </div>_
 
 <br>
 
 ### Viewing all tasks of a patient: `view` `-p`
 
-Shows all the tasks that are associated with the specified patient.
+You can view the list of tasks for a patient using the `view` command.
```
Comment (rama-pang) [inline docs/UserGuide.md:706]: Minor nitpick:
```suggestion
You can view the list of tasks for a particular patient using the `view` command.
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -667,69 +667,74 @@ _Add screenshot here_
...
 
-Shows a list of all tasks to be completed.
+You can view the list of tasks for all patients using the `view` command with the special flag `--all`.
 
 Format: `view -p --all`
 
 Examples:
 
-Suppose the following patients were added.
+* `view -p --all` will show the list of all patients with tasks along their list of tasks.
```
Comment (rama-pang) [inline docs/UserGuide.md:697]: "with tasks along their list of tasks" seems a bit weird, perhaps could be reworded?

Comment (BlopApple) [reply docs/UserGuide.md]: Yup I reworded it. Hopefully it's a bit better now.

**Review by izzahaj — APPROVED** [review]
LGTM

**Pull request merged**